feat(*) document metrics endpoint filtering (#457)
kumahq/kuma-website: docs/docs/1.2.0/policies/traffic-metrics.md

Both snippets from above instruct `Kuma` to configure every dataplane in the mesh `default` to expose an HTTP endpoint with `Prometheus` metrics on port `5670` and URI path `/metrics`.

> The exposed metrics endpoint forwards to the standard Envoy [Prometheus metrics endpoint](https://www.envoyproxy.io/docs/envoy/latest/operations/admin#get--stats?format=prometheus) and supports the same query parameters.
> By default this endpoint returns all available metrics, but if only a subset of the available metrics are needed, the `filter` query parameter can be used to limit the results to metrics whose names match a given regular expression.

Reviewer: ```suggestion
The metrics endpoint is forwarded to the standard Envoy [Prometheus metrics endpoint](https://www.envoyproxy.io/docs/envoy/latest/operations/admin#get--stats?format=prometheus) and supports the same query parameters.
You can pass the `filter` query parameter to limit the results to metrics whose names match a given regular expression. By default all available metrics are returned.
```
